src/routes/docs/products/functions/develop/+page.markdoc

```diff
@@ -1088,7 +1091,6 @@ You can access these logs through the following steps.
 
 # Accessing environment variables {% #environment-variables %} 
 If you need to pass constants or secrets to Appwrite Functions, you can use environment variables. 
-Environmental variables can be global, or function-specific.
```

Comment:
Why remove this?

Reply:
This is not about configuring these variables, just consuming it. No reason to talk about it here, the include steps are the exact same.